README.md

```diff
@@ -1,61 +1,87 @@
+* **Best practices**: the complexity of Challenges means that we need to systematically be able to share patterns and approaches that are demonstrated to work in various situations, from phased approaches through to methods for surfacing up the inherent complexity or coming to an agreed shared understanding.
+* **Governance & Oversight**: being able to manage a portfolio of Challenges, with confidence that agreed governance / processes are being followed. Setting clear principles and boundaries for the collaboration that takes place.
+* **Trust & Transparency**: a neutral, trusted base from which all parties can confidently interact without fear of being at a disadvantage or seeing value capture unfairly by other parties.
+* **Awareness & Visibility**: being able to promote the set of Challenges within a particular context and to share in a controlled way information about both the desired outcomes and the progress towards them.
 
-Realising a platform to facilitate complex challenges is ambitious - but one that can deliver huge value...a step change in how we collaborate!
 
 If you are interested in getting involved please reach out and connect via <info@cherrytwist.org>!
 
-# How is Cherrytwist built? 
-The set of concepts and terms used for the design of CherryTwist can be found in our [Conceptual Design](./Design/ConceptualDesign.md). It is important to read this document first, as a lot of the terminology used in the rest of the design documentation as well as the code is described here - so having this mental model clear helps. 
+<p></p>
+
+
+## How do I get started?
+The easiest way to get started with Cherrytwist is to see it in action! For that there are a couple of options:
+* **[Odyssey Production instance](https://cherrytwist.odyssey.org)** - a production Cherrytwist instance showing Odyssey's hosted Challenges from their Odyssey Momentum event late 2020. 
+* **[UN SDGs Demo](https://demo-sdgs.cherrytwist.org)** - coming soon, a sample Cherrytwist instance illustrating the UN Sustainable Development Goals (SDGs)
+* **[Try it out locally](https://www.github.com/cherrytwist/demo)** - create your own local Cherrytwist instance and populate it with sample data.
 
-A high level technical design for CherryTwist can be found in our [Technical Design](./Design/TechnicalDesign.md).
+## How do I contribute to Cherrytwist?
+There is a huge amount to be done so all contributions are much appreciated! 
 
-A key goal of the platform is to leverage the latest technical capabilities such as Self Sovereign Identity, verified credentials (trust, reputation), formalised agreements via smart contracts etc etc - without exposing those details to the end user!
+Next steps:
+* **[Guidelines for contributing](docs/contributing.md)** 
+* **[Conceptual Design - The key concepts used within the Cherrytwist Platform)](docs/conceptual-design.md)**
```

Comment:
Links in the conceptual design MD are broken. Also a lot of the conceptual design text overlaps with the text on the landing page - needs to be edited.